src/main/java/Helper/Ui.java

```diff
@@ -0,0 +1,161 @@
+package Helper;
+
+/**
+ * This file contains the implementation of Ui class.
+ * @author Saravanan Anuja Harish
+ */
+
+import Tasks.*;
+
+public class Ui {
+
+    /**
+     * prints introductory greetings.
+     */
+    public static void printGreeting() {
+
+        String greeting = "Hello There, My name is Conan! \n"
+                + "Hope you're doing fine today! (^_^) \n"
+                + "I'm a task manager, and I can help you keep up with your tasks.\n"
+                + "Now before we start, lets get acquainted! Lets start with our names!";
+        System.out.println(greeting);
+    }
+
+    /**
+     * prints separator, to differentiate between different commands.
+     */
+    public static void printSeparator() {
+        String separator = "------------------------------------------------";
+        System.out.println(separator);
+    }
+
+    /**
+     * prints a farewell message to the user.
+     *
+     * @param username the name of the user.
+     */
+    public static void printFarewell(String username) {
+        String farewell = "\nHope I helped you complete your tasks!\n"
+                + "Have a great day ahead, enjoy ! (^-^)/\n"
+                + "Hope to see you next time! ";
+        System.out.println("Goodbye, " + username + farewell);
+    }
+
+    /**
+     * prints an introductory message.
+     *
+     * @param username the name of the user.
+     */
+    public static void printSayHello(String username) {
+        String str = "Hello, " + username + "!, Nice to meet you! (*^_^*)\n" +
+                "So, tell me what would you like to do, " + username + "!";
+        System.out.println(str);
+    }
+
+    /**
+     * prints a message saying that a user under a similar name was found.
+     *
+     * @param previousUser the previous username.
+     */
+    public static void printFoundSimilarName(String previousUser) {
```

Comment:
Nice! This is just one example that I wanted to point out. Good to see that many names for methods and variables do not just differentiate between themselves, but also explain what they do.